javascript/features/communication_commands/direct_communication_commands.test.js

```diff
@@ -158,6 +158,18 @@ describe('DirectCommunicationCommands', (it, beforeEach) => {
         assert.equal(gunther.messages.length, 1);  // unchanged
     });
 
+    it('cannot send to muted targets unless sender is an admin', async (assert) => {
+        const playground = server.featureManager.loadFeature('playground');
+        
+        // Set up by Russell muting themselves
+        assert.isTrue(await russell.issueCommand('/mute Russell 5'));
+
+        // (1) It should not be possible for Gunther to PM Russell who's muted
+        assert.isTrue(await gunther.issueCommand('/pm Russell Hello world'));
+        assert.equal(gunther.messages.length, 1);
+        assert.equal(gunther.messages[0],Message.format(Message.COMMUNICATION_PM_TARGET_MUTED, russell.name, russell.id, "5 minutes"));
```

Comment:
nit: we have a line length limit of 100 characters, and please use single quotes.

```javascript
    assert.equal(
        gunther.messages[0],
        Message.format(Message.COMMUNICATION_PM_TARGET_MUTED, russell.name, russell.id,
                       '5 minutes'));
```

Reply:
Fixed as of 8730b90088d43baa26d1f881561c7d54a913bdf6